test/Querier.test.ts

```diff
@@ -14,25 +14,30 @@ const successQuery = async () => {
   const res = await request('/users/1');
   return res;
 };
-const storeMock: QuerierStoreType = {
-  queryKey: {
-    id: 'queryKey',
-    state: {
-      state: QuerierState.Active
-    },
-    result: null,
-    $reason: 'SomeComponent',
-    $props: {}
-  }
+
+const queryKey = 'key';
+const queryResult = users[1];
+const queryReason = 'Some Component';
+
+const createStore = (
+  state: { state: QuerierState } = { state: QuerierState.Success },
+  // tslint:disable-next-line
+  $props: any = null,
+  id: string = queryKey,
+  result: { name: string } = queryResult,
+  $reason: string = queryReason
+): QuerierStoreType => {
+  return { key: { id, result, state, $props, $reason } };
 };
+const storeMock: QuerierStoreType = createStore({ state: QuerierState.Active }, null, null);
 
-describe('Querier', () => {
+fdescribe('Querier', () => {
   describe('utilities', () => {
     test('getEntry', () => {
       const querier = new Querier(storeMock);
 
       expect(Object.keys(querier.getStore())).toHaveLength(1);
-      expect(querier.getEntry('queryKey')).toEqual(storeMock.queryKey);
+      expect(querier.getEntry('key')).toEqual(storeMock.key);
```

Comment:
What about using `queryKey` const here? To be even more verbose, I suggest doing:

```
expect(querier.getEntry(queryKey)).toEqual(storeMock[queryKey]);
```